exonum-java-binding/doc/Migration_guide_0.9.md

```diff
@@ -0,0 +1,179 @@
+# Migrating Java Services to 0.9
+
+The flagship feature of Exonum Java 0.9 release is support for dynamic services&nbsp;—
```

Comment:
does `&nbsp;` really needed?

Reply:
If we want to keep that emdash on the line) Not very much.